pkg/cache/placeholder.go

```diff
@@ -104,3 +108,20 @@ func (p *Placeholder) String() string {
 	return fmt.Sprintf("appID: %s, taskGroup: %s, podName: %s/%s",
 		p.appID, p.taskGroupName, p.pod.Namespace, p.pod.Name)
 }
+
+func getPlaceHolderImage() string {
+	if placeHolderImage == "" {
+		imageEnvSetting := os.Getenv(constants.PlaceHolderImageEnvVar)
```

Comment:
Let's use the standard configuration mechanism we already have in place rather than ENV vars here (those should be handled in the Docker container config and passed as arguments). Be sure to update both the shim and plugin setups.